src/test/java/EnergyTest.java

```diff
@@ -0,0 +1,93 @@
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class EnergyTest {
+
+    @Test
+    void energy4x4() {
+        //given
+        Color color1 = new Color(255, 0, 0);
+        Color color2 = new Color(0, 255, 0);
+        Color color3 = new Color(0, 0, 255);
+
+        Pixel[][] image = new Pixel[][]{
+                {new Pixel(color1.getRGB()),
+                        new Pixel(color2.getRGB()),
+                        new Pixel(color3.getRGB()),
+                        new Pixel(color1.getRGB())},
+                {new Pixel(color2.getRGB()),
+                        new Pixel(color3.getRGB()),
+                        new Pixel(color1.getRGB()),
+                        new Pixel(color1.getRGB())},
+                {new Pixel(color3.getRGB()),
+                        new Pixel(color1.getRGB()),
+                        new Pixel(color1.getRGB()),
+                        new Pixel(color2.getRGB())},
+                {new Pixel(color1.getRGB()),
+                        new Pixel(color1.getRGB()),
+                        new Pixel(color2.getRGB()),
+                        new Pixel(color3.getRGB())}};
```

Comment:
instead of using color1, color2, color3 and calling getRGB over and over again, just replace it with the correct values.